index.php

```diff
@@ -1,146 +1,69 @@
-                          transform="translate(-231.64 -80.34)"/>
-                </svg>
-            </a>
-        </div>
-    </div>
-</footer>
+$layout_content = include_template('layout.php', ['content' => $page_content, 'categories' => $categories, 'title' => $user_name]);
+
+echo($layout_content);
```

Comment:
Важно понимать, что echo – это не функция, поэтому скобки не нужны. Лучше их не использовать с ней вообще.
```suggestion
echo $layout_content;
```